libdrgn/python/thread.c

```diff
@@ -56,6 +56,21 @@ static DrgnObject *Thread_get_object(Thread *self)
 	return_ptr(ret);
 }
 
+static PyObject *Thread_get_name(Thread *self)
+{
+	_cleanup_free_ char *ret = NULL;
+	struct drgn_error *err = drgn_thread_name(&self->thread, &ret);
+	if (err)
+		return set_drgn_error(err);
+	if (!ret) {
+		return Py_None;
+	}
```

Comment:
This needs to be `Py_RETURN_NONE` so that `None` is properly reference-counted on older Python versions where it is not immortal:
```suggestion
	if (!ret)
		Py_RETURN_NONE;
```